### DISCUSSION TITLE
lineage-17.1: Failed issue

### DISCUSSION THREAD

**Discussion by aslenofarid** [ROM-builders/temporary#12716] [Q&A]
always stop here

FAILED: ninja: external/ninja/src/deps_log.cc:631: bool DepsLog::RecordId(Node *): Assertion `node->path().size() > 0' failed.

20:15:23 soong bootstrap failed with: signal: aborted (core dumped)

**Comment by Apon77** [comment]
what can I do?

**Reply by aslenofarid** [reply]
delete lineage-17.1 folder or remove lineage-17.1 support and add it again 🤔

**Reply by Apon77** [reply]
it is done already, it wont solve i think

**Reply by aslenofarid** [reply]
😔😔is there no solution?
